test/onnx/onnx_import_dyn_shapes.in.cpp

```diff
@@ -550,3 +550,11 @@ NGRAPH_TEST(onnx_dyn_shapes_${BACKEND_NAME}, expand_uint16_dyn_shape)
 
     test_case.run();
 }
+
+NGRAPH_TEST(onnx_${BACKEND_NAME}, scatter_elements_import_only)
+{
+    const auto scatter_elements = onnx_import::import_onnx_model(
+        file_util::path_join(SERIALIZED_ZOO, "onnx/scatter_elements.prototxt"));
+    auto test_case =
```

Comment:
Let's remake this test to import the ONNX model and do a few asserts on the resulting nGraph Function.